modules/custom/openy_upgrade_tool/src/OpenyUpgradeLogManager.php

```diff
@@ -0,0 +1,588 @@
+   */
+  protected $configExists = FALSE;
+
+  /**
+   * ConfigImporterService constructor.
+   *
+   * @param \Drupal\features\FeaturesManagerInterface $features_manager
+   *   Features Manager.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   Entity type manager.
+   * @param \Drupal\Core\Config\StorageInterface $config_storage
+   *   The config storage.
+   * @param \Drupal\Core\Config\ConfigManagerInterface $config_manager
+   *   The configuration manager.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   The renderer service.
+   * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $event_dispatcher
+   *   The event dispatcher used to notify subscribers of config import events.
+   * @param \Drupal\Core\Lock\LockBackendInterface $lock
+   *   The lock backend to ensure multiple imports do not occur at same time.
+   * @param \Drupal\Core\Config\TypedConfigManagerInterface $typed_config
+   *   The typed configuration manager.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler.
+   * @param \Drupal\Core\Extension\ModuleInstallerInterface $module_installer
+   *   The module installer.
+   * @param \Drupal\Core\Extension\ThemeHandlerInterface $theme_handler
+   *   The theme handler.
+   * @param \Drupal\Core\Logger\LoggerChannelInterface $loggerChannel
+   *   Logger channel.
+   */
+  public function __construct(
+    FeaturesManagerInterface $features_manager,
+    EntityTypeManagerInterface $entity_type_manager,
+    StorageInterface $config_storage,
+    ConfigManagerInterface $config_manager,
+    ConfigFactoryInterface $config_factory,
+    RendererInterface $renderer,
+    EventDispatcherInterface $event_dispatcher,
+    LockBackendInterface $lock,
+    TypedConfigManagerInterface $typed_config,
+    ModuleHandlerInterface $module_handler,
+    ModuleInstallerInterface $module_installer,
+    ThemeHandlerInterface $theme_handler,
+    LoggerChannelInterface $loggerChannel) {
+
+    $this->featuresManager = $features_manager;
+    $this->logger = $loggerChannel;
+    $this->entityTypeManager = $entity_type_manager;
+    $this->configStorage = $config_storage;
+    $this->renderer = $renderer;
+    // Services necessary for \Drupal\Core\Config\ConfigImporter.
+    $this->eventDispatcher = $event_dispatcher;
+    $this->configManager = $config_manager;
+    $this->configFactory = $config_factory;
+    $this->lock = $lock;
+    $this->typedConfigManager = $typed_config;
+    $this->moduleHandler = $module_handler;
+    $this->moduleInstaller = $module_installer;
+    $this->themeHandler = $theme_handler;
+  }
+
+  /**
+   * Get entity storage that used for upgrade tool.
+   */
+  public function getLoggerEntityStorage() {
+    // TODO: Delete this, after deprecated 'logger_entity' deleting
```

Comment:
it is not deprecated

Reply:
logger_entity not used in Open Y code anymore

Reply:
It is used in OpenY instances, can't be deprecated or removed @hamrant